### DISCUSSION TITLE
How should I make my butttons Vimiumable?

### DISCUSSION THREAD

**Discussion by BWStearns** [philc/vimium#4490] [General]
I'm a longtime vimium user and now I'm developing a site and annoyed that my onClick elements aren't showing up as a links on pressing `f`. Is there a simple way to make elements flagged as clickable for vimium without making them literal links?

**Comment by gdh1995** [comment]
You may add an attribute of `role="button"`.

**Reply by BWStearns** [reply]
Thanks!

**Comment by philc** [comment]
Or use `<button>` as the clickable element.

**Reply by dimitrismv** [reply]
As we obv cannot control how others define clickable elements in their apps, can we define in vimium what counts as a clickable element to be picked up by 'f'. img and svg elements are good candidates maybe that I'd like to try adding in my vimium settings. 

And drop down menus and their items are nearly impossible to be accessed using vimium - unless I'm missing on sth basic.
